src/workflow-context/workflow-provider.js

```diff
@@ -33,7 +36,7 @@ const WorkflowProvider = ({ children }) => {
         return null
     }
 
-    if (loading || !called) {
+    if (!called || fetching) {
```

Comment:
I've dropped `loading` from the condition. The usage is:

`fetching`: will be true whenever data is being fetched
`loading`: will be true if there is no data and the query is currently fetching

You can use loading if you want to use SWR, and fetching if you want to always show a spinner. Loading and fetching together will be the same as just using fetching.